addons/web/static/src/views/fields/daterange/daterange_field.xml

```diff
@@ -6,7 +6,7 @@
             <t t-esc="formattedValue" />
         </t>
         <t t-else="">
-            <input t-ref="root" type="text" t-att-value="formattedValue" t-att-placeholder="props.placeholder" t-on-change="onChangeInput" />
+            <input class="o_input" style="width:100% !important"  t-ref="root" type="text" t-att-value="formattedValue" t-att-placeholder="props.placeholder" t-on-change="onChangeInput" />
```

Comment:
Don't set inline style, but prefer bootstrap classes. Here, you can use w-100, which will do exactly that :)

Reply:
The problem is that I need the `!important` because I override another `width:auto !important`

Reply:
It's not great for sure, but removing completely the oe_inline would be more a scss refactor task, so I guess we can allow it.